src/components/DuosDatePicker.tsx

```diff
@@ -0,0 +1,199 @@
+        main:'#ffffff',
+        contrastText:duosColorBlue
+      },
+    },
+    components: {
+      MuiButton:{
+        styleOverrides:{
+          text:{
+            fontFamily: 'Montserrat',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px',
+          },
+        },
+      },
+      MuiIconButton: {
+        styleOverrides: {
+          sizeMedium: {
+            color: duosColorBlue,
+          },
+        },
+      },
+      MuiDayCalendar:{
+        styleOverrides:{
+          weekDayLabel:{
+            fontFamily: 'Montserrat',
+            color: 'black',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px',
+            '&:first-of-type': {
+              color: 'red',
+            },
+            '&:last-of-type': {
+              color: 'red',
+            },
+          },
+        },
+      },
+      MuiPickersArrowSwitcher:{
+        styleOverrides:{
+          button:{
+            fontSize:'2.5rem',
+          },
+        },
+      },
+      MuiPickersDay: {
+        styleOverrides: {
+          root: {
+            '--weekend': 'red',
+            borderRadius: '8px',
+            fontFamily: 'Montserrat',
+            color: 'black',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px',
+            '&.Mui-selected': {
+              backgroundColor: duosColorBlue,
+              '&:hover': {
+                backgroundColor: duosColorBlue,
+              },
+              '&:focus': {
+                backgroundColor: duosColorBlue,
+              },
+            },
+          },
+        },
+      },
+      MuiPickersYear: {
+        styleOverrides: {
+          yearButton: {
+            fontFamily: 'Montserrat',
+            color: 'black',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px',
+            '&.Mui-selected': {
+              backgroundColor: duosColorBlue,
+              '&:hover': {
+                backgroundColor: duosColorBlue,
+              },
+              '&:focus': {
+                backgroundColor: duosColorBlue,
+              },
+            },
+          },
+        },
+      },
+      MuiPickersCalendarHeader: {
+        styleOverrides: {
+          root: {
+            fontSize:'2.5rem',
+          },
+          switchViewIcon:{
+            fontSize:'2.5rem',
+          },
+          label:{
+            fontFamily: 'Montserrat',
+            color: 'black',
+            fontSize: '13px',
+            fontWeight: '400',
+            padding: '7px 20px',
+          },
+        }
+      },
+    },
+  });
+
+  const CancelSelectActionBar = (props:PickersActionBarProps) => {
+    // Quirk of this control's usage pattern is the need to destructure the unused onSetToday and onClear from 'other'
+    // props.  This is in part because per mockup, this control does not support 'clear' or 'go to today' style buttons.
```

Comment:
can you link the mocks? we don't need a clear button because there is cancel? and we don't need go to today because today is the default value? is that right?